Spigot-API-Patches/0167-Add-more-Evoker-API.patch

```diff
@@ -0,0 +1,43 @@
+From 7ac07ac07ac07ac07ac07ac07ac07ac07ac07ac0 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <Blake.Galbreath@GMail.com>
+Date: Fri, 26 Oct 2018 02:39:21 -0500
+Subject: [PATCH] Add more Evoker API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Evoker.java b/src/main/java/org/bukkit/entity/Evoker.java
+index 7ac07ac07ac0..7ac07ac07ac0 100644
+--- a/src/main/java/org/bukkit/entity/Evoker.java
++++ b/src/main/java/org/bukkit/entity/Evoker.java
+@@ -1,5 +1,7 @@
+ package org.bukkit.entity;
+ 
++import javax.annotation.Nullable; // Paper
++
+ /**
+  * Represents an Evoker "Illager".
+  */
+@@ -60,4 +62,21 @@ public interface Evoker extends Spellcaster {
+      */
+     @Deprecated
+     void setCurrentSpell(Spell spell);
++
++    // Paper start
++    /**
++     * Get the sheep being targeted by the Wololo spell
++     *
++     * @return Targeted sheep
++     */
++    @Nullable
++    Sheep getWololoTarget();
```

Comment:
I question if we should make this return Mob instead of sheep incase mojang changes the behavior of this spell in the future?

Reply:
We can break API if they change it.